components/calendar/index.css

```diff
@@ -281,8 +352,186 @@ governing permissions and limitations under the License.
+      }
+    }
+
+    &.is-range-selection {
+      &:before {
+        background: var(--spectrum-calendar-day-background-color-selected-hover, var(--spectrum-calendar-day-background-color-selected-hover));
+      }
+    }
+
+    &:before {
+      border-color: var(--mod-calendar-day-border-color-key-focus, var(--spectrum-calendar-day-border-color-key-focus));
+    }
+  }
+
+  &.is-disabled {
+    color: var(--mod-calendar-day-text-color-disabled, var(--spectrum-calendar-day-text-color-disabled));
+  }
+
+  &.is-selection-start,
+  &.is-selection-end {
+    color: var(--mod-calendar-day-text-color-cap-selected, var(--spectrum-calendar-day-text-color-cap-selected));
+
+    &:after {
+      background-color: var(--mod-calendar-day-background-color-selected, var(--spectrum-calendar-day-background-color-selected));
+    }
+
+    &.is-disabled {
+      color: var(--mod-calendar-day-text-color-disabled, var(--spectrum-calendar-day-text-color-disabled));
+    }
+  }
+}
+
+@media (forced-colors: active) {
+  .spectrum-Calendar-date {
+    color: CanvasText;
+    forced-color-adjust: none;
+    --spectrum-calendar-button-icon-color: ButtonText;
+    --spectrum-calendar-day-background-color-cap-selected-down: Highlight;
+    --spectrum-calendar-day-background-color-cap-selected-hover: Highlight;
+    --spectrum-calendar-day-background-color-cap-selected: Highlight;
+    --spectrum-calendar-day-background-color-disabled: ButtonFace;
+    --spectrum-calendar-day-background-color-down: ButtonFace;
+    --spectrum-calendar-day-background-color-hover: Transparent;
+    --spectrum-calendar-day-background-color-key-focus: ButtonFace;
+    --spectrum-calendar-day-background-color-selected-hover: Transparent;
+    --spectrum-calendar-day-background-color-selected: Highlight;
+    --spectrum-calendar-day-border-color-key-focus: ButtonText;
+    --spectrum-calendar-day-text-color-cap-selected-hover: HighlightText;
+    --spectrum-calendar-day-text-color-cap-selected: HighlightText;
+    --spectrum-calendar-day-text-color-disabled: GrayText;
+    --spectrum-calendar-day-text-color-hover: ButtonText;
+    --spectrum-calendar-day-text-color-key-focus: ButtonText;
+    --spectrum-calendar-day-text-color-selected-hover: HighlightText;
+    --spectrum-calendar-day-text-color-selected: HighlightText;
+    --spectrum-calendar-day-title-text-color: CanvasText;
+    --spectrum-calendar-day-today-background-color-down: ButtonText;
+    --spectrum-calendar-day-today-background-color-hover: ButtonText;
+    --spectrum-calendar-day-today-background-color-selected-hover: Highlight;
+    --spectrum-calendar-day-today-background-color: ButtonText;
+    --spectrum-calendar-day-today-border-color-disabled: GrayText;
+    --spectrum-calendar-day-today-border-color: ButtonText;
+    --spectrum-calendar-day-today-text-color-disabled: GrayText;
+    --spectrum-calendar-day-today-text-color: ButtonText;
```

Comment:
I would update these to the `--highcontrast-` variable names for to ensure that they are used and override any mods